syntax = "proto3";

message LocationMessage {
  int32 person_id= 1;
  float longitude = 2;
  float latitude = 3;
  int64 creation_time = 4; // timestamp will be in epoch type
}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
    // Push received info into a kafka topic
}